src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -282,17 +305,53 @@ export default {
 		.instant-search-container {
 			width: 100%;
 
-			.loan-card-group {
-				opacity: 1;
-				transition: opacity $filter-transition;
+			.lend-filter-results-container {
+				.algolia-search-box-component {
+					margin-top: 1.125rem;
+				}
+
+				.loan-card-group {
+					opacity: 1;
+					transition: opacity $filter-transition;
 
-				&.filter-menu-open {
-					opacity: 0.2;
+					&.filter-menu-open {
+						opacity: 0.2;
+					}
 				}
 			}
 		}
 
 		&.filter-menu-pinned {
+			@include breakpoint(1193px down) {
```

Comment:
We needed to rearrange `SelectedRefinements` when in the pinned filters component and not on the largest breakpoint